caom2repo/caom2repo/core.py

```diff
@@ -89,13 +89,17 @@
 from six.moves.urllib.parse import urlparse
 
 # from . import version as caom2repo_version
-from . import version
+from caom2repo import version
```

Comment:
I think this change  will grab the version of the caom2repo that is in the PythonPath which might not be the version of the caom2repo that this core.py goes with.  Is that the desire? It add ambiguity.